fdm-app/app/lib/auth-client.ts

```diff
@@ -0,0 +1,15 @@
+import {
+    createAuthClient
```

Comment:
you might skip this one if not used

Reply:
I have checked and found that at signIn it is actually used, so it cannot be removed